static/app/types/profiling.d.ts

```diff
@@ -0,0 +1,67 @@
+namespace Profiling {
+  interface RawProfileBase {
+    endValue: number;
+    startValue: number;
+    name: string;
+    unit: string;
+    spans?: RawSpan[];
+  }
+
+  // Android traces follow this format
+  interface EventedProfile extends RawProfileBase {
+    events: ReadonlyArray<{at: number; frame: number; type: 'O' | 'C'}>;
+    type: 'evented';
+  }
+
+  // iOS traces follow this format
+  interface SampledProfile extends RawProfileBase {
+    weights: number[];
+    samples: number[][];
+    type: 'sampled';
+  }
+
+  type Span = {
+    duration_ms: number;
+    name: string;
+    queue_label: string;
+    relative_start_ms: number;
+    thread_id: number;
+    children?: Span[];
+  };
+
+  type FrameInfo = {
+    key: number | string;
+    name: string;
+    file?: string;
+    line?: number;
+    column?: number;
+    is_application?: boolean;
+    image?: string;
+    resource?: string;
+  };
+
+  type ProfileTypes =
+    | EventedProfile
+    | SampledProfile
+    | JSSelfProfiling.SpectoProfilerTrace;
```

Comment:
@JonasBa Where is `JSSelfProfiling.SpectoProfilerTrace` defined?

Reply:
@dashed this is a typo from my side. Should be `JSSelfProfiling.Trace`. Do you know why the CI doesnt fail here, feels like it should scream here 🤔